BDArmory/Control/_description

```diff
@@ -1,2 +1,2 @@
 Control logic for vessels.
-FIXME AI and WM modules should probably be moved here along with various other modules, e.g., WingCommander.
\ No newline at end of file
+FIXME AI and WM modules should probably be moved here along with various other modules, e.g., WingCommander.
```

Comment:
if the modular missile guidance is here, might make sense to move missilebase as well, since they basically do the same things

Reply:
Hmm, maybe. I figured the MMG was closer to the AI's as that's kinda how they're used in-game, but looking at the code, I see it's functionally more like the missilebase, so then it's probably better suited to go in the Weapons category, possibly with a missiles sub-category since there are several missile-specific files there.